primitives/traits/src/lib.rs

```diff
@@ -219,6 +225,15 @@ pub trait RemoveInvulnerables<AccountId> {
     ) -> Vec<AccountId>;
 }
 
+impl<AccountId: Clone> RemoveInvulnerables<AccountId> for () {
+    fn remove_invulnerables(
+        collators: &mut Vec<AccountId>,
+        _num_invulnerables: usize,
+    ) -> Vec<AccountId> {
+        collators.clone()
```

Comment:
The `RemoveInvulnerables` is a bit awkward to use, but you are expected to remove items from the `collators` vec. So a better implementation would be to return `vec![]`, which would indicate that there are no invulnerables. That's not correct in starlight (because all collators are invulnerables) but since this trait is used to implement the invulnerables priority feature, it shouldn't matter.
```suggestion
        // Default impl: no collators are invulnerables
        vec![]
```

Reply:
So to me `RemoveInvulnerables = ()` should in fact indicate that I dont want any changes, you think it's not descriptive enough?